lib/fakefs/file.rb

```diff
@@ -197,11 +197,11 @@ def self.read(path, *args)
       file.read(length)
     end
 
-    def self.readlines(path)
+    def self.readlines(path, options = {})
```

Comment:
kw args for better error messages ?
```suggestion
    def self.readlines(path, chomp: false)
```

Reply:
@grosser Done in 7fd1d6e! Thanks for the suggestion 😁